src/Plugin.Maui.Audio/IAudioPlayer.cs

```diff
@@ -30,6 +30,22 @@ public interface IAudioPlayer : IDisposable
 	///</Summary>
 	double Balance { get; set; }
 
+	///<Summary>
+	/// Gets or sets the playback speed where 1 is normal speed.
+	///</Summary>
+	///<remarks>
+	/// The minimum and maximum speeds that can be set here are different per platform. Setting values ouside of these ranges will not throw an exception, it will clamp to the minimum or maximum value.
+	///<para>- Android: between 0 and 6. Setting the value to 0 will pause playback, playback will not be resumed when incrementing the value again.</para>
+	///<para>- iOS: between 0.5 and 2.</para>
+	///<para>- Windows: between 0 and 8. Setting the value to 0 will pause playback, playback will be resumed when incrementing the value again.</para>
+	///</remarks>
+	double Speed { get; set; }
+
+	///<Summary>
+	/// Gets a value indicating whether the playback speed can be changed.
+	///</Summary>
+	bool CanSetSpeed { get; }
```

Comment:
Should we add `double MinimumSpeed` and `double MaximumSpeed` properties that each platform will return? Then the sample app could use those values for the max/min values of the `Slider`?

Reply:
Ooh I love that idea! Added!